m4/ax_check_compile_flag.m4

```diff
@@ -0,0 +1,53 @@
+# ===========================================================================
```

Comment:
Did you force git to add this file? I think this is currently matched by `.gitignore` rules and should be explicitly excluded

Reply:
Done.